startup/98-suspenders.py

```diff
@@ -56,7 +56,7 @@
 RE.install_suspender(ring_suspender)
 #RE.install_suspender(solenoid_v_suspenderCeil)
 #RE.install_suspender(solenoid_v_suspenderFloor)
-RE.install_suspender(I0_suspenderFloor)
+#RE.install_suspender(I0_suspenderFloor)
```

Comment:
Should be optionally turned off via the environment variable `AUTOALIGNMENT` (or a better name).
```suggestion
#RE.install_suspender(I0_suspenderFloor)
```